src/test/__tests__/ReactDefaultPerf-test.js

```diff
@@ -24,7 +24,7 @@ describe('ReactDefaultPerf', function() {
 
   beforeEach(function() {
     var now = 0;
-    jest.setMock('performanceNow', function() {
+    jest.setMock('fbjs/lib/performanceNow', function() {
```

Comment:
I don't think you wanted to do this. I'm guessing this will make the test start failing internally since that won't resolve. We should be rewriting this with the fbjs-scripts plugin...